### PR TITLE
[WIP] some small work on the docs

### PR DESCRIPTION
**Pull request by leggewie**
Comments:
- The top-most patch 8b9044f4f is optional (well, all of them are at your discretion, of course).  That's why it's on top to drop it most easily if necessary.
- not sure if the \<tt\>-tags in cbdee6e696 are correct or if one should use apostrophes instead (in case you follow a certain style)

### CODE REVIEW

**Comment by GawainLynch** [comment]
Aeesome mate, thanks! :+1:

**Comment by leggewie** [comment]
ready to merge?

File: doc/quick-start.md [hunk 1]
```diff
@@ -49,6 +50,14 @@ providers:
 ```
 
 
+#### Configure outgoing mail
+
+You should have already done this as part of your inital setup of bolt.  But
```
Comment (GawainLynch) [inline doc/quick-start.md]: Could you be so awesome as to use 'B' for Bolt in cases where is is used for branding :smile:

**Comment by leggewie** [comment]
On 14.02.2017 06:46, Gawain Lynch wrote:
>
> Could you be so awesome as to use 'B' for Bolt in cases where is is
> used for branding 😄
>

Absolutely. Thanks for the review.

**Comment by bobdenotter** [comment]
Thank you! Merging in!

**Pull request merged**